homeassistant/components/light/__init__.py

```diff
@@ -98,6 +98,7 @@
 LIGHT_TURN_OFF_SCHEMA = vol.Schema({
     ATTR_ENTITY_ID: cv.entity_ids,
     ATTR_TRANSITION: VALID_TRANSITION,
+    ATTR_EFFECT: vol.In([EFFECT_COLORLOOP, EFFECT_RANDOM, EFFECT_WHITE]),
```

Comment:
Turn off schema should have `vol.Optional(ATTR_FLASH): vol.In(FLASH_LONG, FLASH_SHORT)` right??